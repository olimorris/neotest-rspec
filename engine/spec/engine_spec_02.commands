:lua require("neotest-rspec")({engine_support=false})
:1
:lua require("neotest").run.run()
