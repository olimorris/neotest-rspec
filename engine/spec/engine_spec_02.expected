./ENGINE/SPEC/ENGINE_SPEC.RB@@FAILED@@runs specs from the engine root
